hyperspy/signal_tools.py

```diff
@@ -930,6 +930,14 @@ def find(self, back=False):
                 # This is only available for traitsui, ipywidgets has a
                 # progress bar instead.
                 pass
+            except ValueError as error:
```

Comment:
I would simply always add the error message to the logger as this check is weak: if we ever change the text of the error message it'll get broken.